src/biogeophys/SoilHydrologyInitTimeConstMod.F90

```diff
@@ -211,7 +211,7 @@ subroutine SoilHydrologyInitTimeConst(bounds, soilhydrology_inst)
                 ! do nothing
              else
                 do lev = 1,nlevgrnd
-                   if ( soil_layerstruct /= '10SL_3.5m' )then
+                   if ( soil_layerstruct_predefined /= '10SL_3.5m' )then
```

Comment:
I see a few clues and have come to no conclusions so will need input from you guys:
1) The same if-statement a few lines up makes reference to VIC, which suggests that VIC cannot use the predefined `10SL_3.5m` option:
`if ( soil_layerstruct_predefined /= '10SL_3.5m' ) write(iulog,*) 'Setting clay, sand, organic, in Soil Hydrology for VIC'`
2) Next, the if-statement that you point out here repeats the code of the second instance that you point out in SoilStateInitTimeConstMod.F90. This seems problematic for code maintenance. Replace repeating code with a subroutine or function?
3) In the second instance, DML (is that you @dlawrenncar ?) inserted the comment:
`! DML - this if statement could probably be removed and just the top part used for all soil layer structures`
I'm not sure if this means eliminating the else `! duplicate clay and sand values from 10th soil layer` section or something different...
4) Last clue, the predefined `10SL_3.5m` option is clm4.5's default currently.

Reply:
@swensosc - I also wanted to ping you here. Do you know anything about the above block of code (i.e., the logic that does something different for 10SL_3.5m, maybe related to VIC), and how it might need to be generalized now that we allow user-defined soil layer structures?

@slevisconsulting if this tag gets close to complete, and this is the last thing to resolve, I'm okay with this being left unresolved for now, until we're able to all meet and discuss it.